### DISCUSSION TITLE
Difference between PartitionExplainer and PermutationExplainer with Partition Masking

### DISCUSSION THREAD

**Discussion by abarbosa94** [shap/shap#3220] [Q&A]
Reading through the documentation, I found that it is possible to use Permutation Explainer with Partition masking, which leads to Owen Values approximations:
```
masker = shap.maskers.Partition(X_train, clustering="correlation")
explainer = shap.explainers.Permutation(model=clf.predict_proba, 
                                 data=X_train, 
                                 masker=masker,
                                 feature_names=["Feature 0", "Feature 1"])
shap_values = explainer(X_test[:1])
```

What I would like to understand is what is the difference between doing this and directly using PartitionExplainer, which seems to be responsible for computing Owen Values of the feature impact

```
masker = shap.maskers.Partition(X_train, clustering="correlation")
explainer = shap.explainers.**Partition**(model=clf.predict_proba, 
                                 data=X_train, 
                                 masker=masker,
                                 feature_names=["Feature 0", "Feature 1"])
shap_values = explainer(X_test[:1])
```

If PartitionExplainer computes Exact Owen Values, then what is the difference between it and using ExactExplainer?

**Comment by detrin** [comment]
I don't see difference between first and second example, except the bold naming

**Reply by abarbosa94** [reply]
@detrin this is exactly my question :)

Algorithmically speaking, what is the difference between PermutationExplainer and PartitionExplainer? Based on the API naming, I'm assuming that both are two different ways to approximate Shapley values. If so, what is the difference between them?
